sklearn/inspection/tests/test_partial_dependence.py

```diff
@@ -476,6 +499,7 @@ def test_partial_dependence_X_list(estimator):
 
 # TODO: Remove in 0.24 when DummyClassifier's `strategy` default updates
 @ignore_warnings(category=FutureWarning)
```

Comment:
Could we catch specifically the warning link to the dummy classifier instead of a global ignore_warnings

Reply:
Done in https://github.com/scikit-learn/scikit-learn/pull/16619/commits/5c3b713561a34ceb3afbff2d2553bec3e849f26d